golem-cli/src/command/component.rs

```diff
@@ -703,7 +704,10 @@ impl ApplicationComponentContext {
             .application_context
             .application
             .component_wasm_rpc_dependencies(component_name)
-            .clone();
+            .iter()
```

Comment:
I actually did not do this intentionally. But ok to change it.

(It does not really matter because the dynamic linker is only handling the interfaces that were not linked statically)

Reply:
it was confusing for me that it was adding static deps into the dynamic_linking field

Reply:
especially not that we show them in the cli